### PR TITLE
Schedule Frequency as JSON

### PR DESCRIPTION
**Pull request by Alex-Tideman**
## Description & motivation 💭 
Due to the infinite combination of possible schedules, instead of trying and failing to use a human readable label, show the calendar and interval as JSON. 

Rearrange layout of table row and Schedule view to make the JSON easily viewable and not take up too much space.

### Screenshots (if applicable) 📸 
Screen Shot 2023-08-29 at 2 25 43 PM
Screen Shot 2023-08-29 at 2 27 43 PM
Screen Shot 2023-08-29 at 2 27 54 PM

### How was this tested 👻 

- [x] Manual testing
- [ ] E2E tests added
- [ ] Unit tests added

### CODE REVIEW

**Comment by yiminc** [comment]
Instead of use 'Frequency', should just be 'Schedule Spec'.

File: src/lib/components/schedule/schedules-table-row.svelte [hunk 1]
```diff
@@ -87,6 +83,17 @@
     {/each}
   </td>
 </TableRow>
+<TableRow>
```
Comment (laurakwhit) [inline src/lib/components/schedule/schedules-table-row.svelte:86]: Do users need to see the entire `Schedule Spec` in this view? Or would even something smaller like this be adequate? 

Screenshot 2023-08-30 at 11 58 01 AM

Or maybe we could add a `!border-t-0` to remove the top border so the row is more visually connected? 

Screenshot 2023-08-30 at 12 01 51 PM

**Review by laurakwhit — APPROVED** [review]
Just some comments r.e. potential UI improvements, otherwise lgtm!

File: src/lib/components/schedule/schedule-frequency.svelte [hunk 2]
```diff
@@ -1,28 +1,20 @@
...
 
   export let calendar: StructuredCalendar | undefined = undefined;
   export let interval: IntervalSpec | undefined = undefined;
-
-  const intervalSecs = interval?.interval as string;
-  const phaseSecs = interval?.phase as string;
+  export let inline = false;
 </script>
 
 <div class="flex flex-col {$$props.class}">
```
[the remark this answers is not in the document]
Comment (Alex-Tideman) [reply src/lib/components/schedule/schedule-frequency.svelte:12]: Nah I want to show the whole thing, and with this layout we get quite a bit of height on the right side so there's no reason to not take it up.

File: src/lib/components/schedule/schedule-frequency-panel.svelte [hunk 3]
```diff
@@ -2,18 +2,17 @@
...
-  
-  
+
   import type { IntervalSpec } from '$types';
 
   export let calendar: StructuredCalendar | undefined = undefined;
   export let interval: IntervalSpec | undefined = undefined;
 </script>
 
 <Panel>
```
[the remark this answers is not in the document]
Comment (Alex-Tideman) [reply src/lib/components/schedule/schedule-frequency-panel.svelte:14]: Yeah makes sense

**Pull request merged**